components/featuredProjects/ProjectCard.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import { Card } from "@nextui-org/react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { arrow } from "@/public";
+
+type projectProps = {
```

Comment:
The type definition projectProps should follow the TypeScript conventions of PascalCase for type and interface names. The correct convention is `ProjectProps`